api/v1/constructors.go

```diff
@@ -981,12 +981,14 @@ func parseFileSystemInfo(spec *SystemSpec, fileSystems []controllerFilesystems.F
 	result := make([]ControllerFileSystemInfo, 0)
 
 	for _, fs := range fileSystems {
-		info := ControllerFileSystemInfo{
-			Name: fs.Name,
-			Size: fs.Size,
-		}
+		if fs.Name == "backup" || fs.Name == "database" {
```

Comment:
We will need to configure other fs types if they are specified in deployment-config.yaml

Reply:
I have added the fs types filter